src/uniswap-sdk/uniswap.ts

```diff
@@ -29,19 +32,31 @@ export const getUniswapRouterContract = (provider: provider) => {
   return contract
 }
 
+const getSushiswapRouterContract = (provider: provider) => {
+  const web3 = new Web3(provider)
+  const contract = new web3.eth.Contract(
+    (UniswapRouterABI.abi as unknown) as AbiItem,
+    sushiswapRouterAddress
+  )
+  return contract
+}
+
 export const getUniswapTradeTransaction = (
```

Comment:
Note: maybe we generalize this a bit since it looks like we're going to be working a bit more with Sushi as time goes on?

Maybe that's worth doing in another PR, and can include the filename

Reply:
yup, sounds good